python/bench/VerificationBenchmarkTemplate.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/python
+#-*- coding: utf-8 -*-
+#===========================================================
+#  File Name: VerificationBenchmarkTemplate.py
+#  Author: Alex Butenko, Georgia Institute of Technology
+#  Creation Date: 05-08-2019
+#  Last Modified: Thu May 23 21:45:20 2019
+#
+#  Description: Standard geometric verification benchmark template
+#
+#  Copyright (C) 2019 Alex Butenko
+#  All rights reserved.
+#
+#  This file is made available under
+#  the terms of the BSD license (see the COPYING file).
+#===========================================================
+
+"""
+This module describe a verification benchmark template.
+A benchmark is given a verification algorithm and a dataset
+"""
+
+import numpy as np
+from abc import ABCMeta, abstractmethod
+import os
+from tqdm import tqdm
+import pickle as pkl
+
+class VerificationBenchmark():
+    __metaclass__ = ABCMeta
+
+    """Verification Benchmark Template
+
+    Attributes
+    ----------
+
+    name: str
+        Name of the benchmark
+    result_dir: str
+        Directory for saving the final result
+    """
+
+    def __init__(self, name,
+                 result_dir='./python_scores/'):
+        self.name = name
+        self.result_dir = result_dir
+
+
+
+    def find_F_matrix(self, verifier, data_dict):
```

Comment:
would be cool to add Python typing to help with readability, like here: https://github.com/argoai/argoverse-api/blob/master/argoverse/utils/cv2_plotting_utils.py#L21